src/main/java/study/step2/domain/WinningTable.java

```diff
@@ -0,0 +1,38 @@
+package study.step2.domain;
+
+import java.util.stream.Stream;
+
+public enum WinningTable {
+    NOTHING(0,0),
+    FOURTH(3, 5000),
+    THIRD(4, 50000),
+    SECOND(5, 1500000),
+    FIRST(6, 2000000000);
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000);
```
위와 같이 가독성을 향상 시킬 수 있어요.